fog/ledger/server/tests/router_integration.rs

```diff
@@ -433,6 +364,13 @@ async fn smoke_test() {
             response.results[0].timestamp_result_code,
             TimestampResultCode::TimestampFound as u32
         );
+        // The genesis block will add 1 to the total number of blocks, it has
+        // one transaction which will also add 1 to the total number of
+        // transactions
+        assert_eq!(response.num_blocks, blocks_to_add + 1);
+        assert_eq!(response.global_txo_count, new_transactions + 1);
+        assert_eq!(response.latest_block_version, *BlockVersion::MAX);
+        assert_eq!(response.max_block_version, *BlockVersion::MAX);
```

Comment:
This verification was missing and is what was exposed when the shards didn't scale up.
The clients rely on these values to know the status, https://github.com/mobilecoinofficial/MobileCoin-Swift/blob/70e92f44dd88d7b7cd4ac6175e188b2acb895217/Sources/Common/Fog/FogKeyImageChecker.swift#L79

The intent is that a follow on PR will update the tests to add one block at a time and ensure that this response has info for the newest block after the addition of each block.